ic10emu/src/vm/object/errors.rs

```diff
@@ -0,0 +1,38 @@
+use serde_derive::{Deserialize, Serialize};
+use thiserror::Error;
+
+use stationeers_data::enums::script::{LogicSlotType, LogicType};
+#[cfg(feature = "tsify")]
+use tsify::Tsify;
+#[cfg(feature = "tsify")]
+use wasm_bindgen::prelude::*;
+
+#[derive(Error, Debug, Clone, Serialize, Deserialize)]
+#[cfg_attr(feature = "tsify", derive(Tsify))]
+#[cfg_attr(feature = "tsify", tsify(into_wasm_abi, from_wasm_abi))]
```

Comment:
this is mostly just styling preference, but it's possible to do this like. I have no preference


```suggestion
#[cfg_attr(feature = "tsify", derive(Tsify), tsify(into_wasm_abi, from_wasm_abi))]
```